src/models/Customer.js

```diff
@@ -1,18 +1,14 @@
 const mongoose = require('mongoose')
 
 const CustomerSchema = new mongoose.Schema({
-  name: {
-    type: String,
-    required: true,
-  },
-  phone: {
-    type: Number,
-    required: true,
-  },
-  age: {
-    type: Number,
-    required: false
-  },
```

Comment:
i deleted the age entity cause i found it a disgusting useless
and I wanna change the check entity name to receipt because the Head of Department told me to do that :(